.gitignore

```diff
@@ -14,3 +14,4 @@
 /Licenses.toml
 /licenses
 *.run
+/tests
```

Comment:
If there's any chance of larger files ending up here, it might be worth adding to `.dockerignore` as well.